include/crud-terms.php

```diff
@@ -307,27 +307,25 @@ private function get_slug_separator() {
 	 * @return string Slug with a language suffix if found.
 	 */
 	public function set_pre_term_slug( $slug, $taxonomy ) {
-		if ( ! empty( $slug ) ) {
-			return $slug;
+		if ( ! $slug ) {
+			$slug = sanitize_title( $this->pre_term_name );
 		}
 
-		$name = sanitize_title( $this->pre_term_name );
-
 		/**
 		 * Filters the subsequently inserted term language.
 		 *
 		 * @since 3.3
 		 *
 		 * @param PLL_Language|null $lang     Found language object, null otherwise.
 		 * @param string            $slug     Term slug
-		 * @param string            $taxonomy Term taxonomy.
+		 * @param string            $taxonomy Term taonomy.
```

Comment:
We could at least fix this typo 😅
```suggestion
		 * @param string            $taxonomy Term taxonomy.
```